Formula/zsh-vi-mode.rb

```diff
@@ -0,0 +1,27 @@
+class ZshViMode < Formula
+  desc "Better and friendly vi(vim) mode plugin for ZSH"
+  homepage "https://github.com/jeffreytse/zsh-vi-mode"
+  url "https://github.com/jeffreytse/zsh-vi-mode/archive/refs/tags/v0.8.2.tar.gz"
+  sha256 "888d68b451fc3221c2ee1648dbfbff38e524048b2c78620a244650fc76946c28"
+  license "MIT"
+
+  bottle :unneeded
+
+  def install
+    pkgshare.install "zsh-vi-mode.zsh"
```

Comment:
Why install this into `pkgshare`, and not the `prefix`? We should probably install `zsh-vi-mode.plugin.zsh` too.

Reply:
I followed install in zsh-autosuggestions.rb and zsh-history-substring-search.rb
yes, adding zsh-vi-mode.plugin.zsh to the install